### PR TITLE
"target" flag descriptions

### PR DESCRIPTION
**Pull request by allaVolkov**
"target" flag descriptions

### Checklist
- [x] Code compiles correctly
- [ ] Relevant tests were added (unit / contract / integration)
- [ ] Relevant logs were added
- [x] Formatting and linting run locally successfully
- [x] All tests pass
- [x] UA review
- [ ] Design is documented
- [ ] Extended the README / documentation, if necessary
- [ ] Open source is approved

### CODE REVIEW

File: cmd/assembly.go [hunk 1]
```diff
@@ -35,9 +35,9 @@ var assembleCommand = &cobra.Command{
 
 func init() {
 	assembleCommand.Flags().StringVarP(&assembleCmdSrc,
-		"source", "s", "", "The path to the MTA project; the current path is set as the default")
+		"source", "s", "", "The path to the MTA project; the current path is set as default")
 	assembleCommand.Flags().StringVarP(&assembleCmdTrg,
-		"target", "t", "", "The path to the results folder; the current path is set as the default")
+		"target", "t", "", `The path to the generated MTAR file; the path to the "mta_archives" subfolder of the current folder is set as default`)
```
Comment (tal-sapan) [inline cmd/assembly.go]: Isn't it the path to the folder in which the MTAR file is created?

File: cmd/gen.go [hunk 2]
```diff
@@ -58,9 +58,9 @@ func init() {
 
 	// set flags of mtar command
 	mtarCmd.Flags().StringVarP(&mtarCmdSrc, "source", "s", "",
-		"The path to the MTA project; the current path is set as the default")
+		"The path to the MTA project; the current path is set as default")
 	mtarCmd.Flags().StringVarP(&mtarCmdTrg, "target", "t", "",
-		"The path to the results folder; the current path is set as the default")
+		`The path to the generated MTAR file; the path to the "mta_archives" subfolder of the current folder is set as default`)
```
Comment (tal-sapan) [inline cmd/gen.go]: Same - isn't it the path to the folder in which the MTAR file is created?

File: cmd/init.go [hunk 3]
```diff
@@ -35,16 +35,16 @@ var buildCmdOutputSync bool
...
 	initCmd.Flags().StringSliceVarP(&initCmdExtensions, "extensions", "e", nil, "The MTA extension descriptors")
 	initCmd.Flags().StringVarP(&initCmdMode, "mode", "m", "", `The mode of the Makefile generation; supported values: "default" and "verbose"`)
 	_ = initCmd.Flags().MarkHidden("mode")
 	initCmd.Flags().BoolP("help", "h", false, `Displays detailed information about the "init" command`)
 
 	// set flags of build command
-	buildCmd.Flags().StringVarP(&buildCmdSrc, "source", "s", "", "The path to the MTA project; the current path is set as the default")
-	buildCmd.Flags().StringVarP(&buildCmdTrg, "target", "t", "", "The path to the results folder; the current path is set as the default")
+	buildCmd.Flags().StringVarP(&buildCmdSrc, "source", "s", "", "The path to the MTA project; the current path is set as default")
+	buildCmd.Flags().StringVarP(&buildCmdTrg, "target", "t", "", `The path to the generated MTAR file; the path to the "mta_archives" subfolder of the current folder is set as default`)
```
Comment (tal-sapan) [inline cmd/init.go]: Same - isn't it the path to the folder in which the MTAR file is created?

File: cmd/assembly.go (hunk 1 above)
Comment (allaVolkov) [reply cmd/assembly.go]: changed

File: cmd/gen.go (hunk 2 above)
Comment (allaVolkov) [reply cmd/gen.go]: changed

File: cmd/assembly.go [hunk 4]
```diff
@@ -35,9 +35,9 @@ var assembleCommand = &cobra.Command{
 
 func init() {
 	assembleCommand.Flags().StringVarP(&assembleCmdSrc,
-		"source", "s", "", "The path to the MTA project; the current path is set as the default")
+		"source", "s", "", "The path to the MTA project; the current path is set as default")
 	assembleCommand.Flags().StringVarP(&assembleCmdTrg,
-		"target", "t", "", "The path to the results folder; the current path is set as the default")
+		"target", "t", "", `The path to the folder in which MTAR file is created; the path to the "mta_archives" subfolder of the current folder is set as default`)
```
Comment (tal-sapan) [inline cmd/assembly.go]:
```suggestion
		"target", "t", "", `The path to the folder in which the MTAR file is created; the path to the "mta_archives" subfolder of the current folder is set as default`)
```
(For the 2 other places with this string as well)

File: cmd/init.go [hunk 5]
```diff
@@ -35,16 +35,16 @@ var buildCmdOutputSync bool
 
 func init() {
 	// set flags for init command
-	initCmd.Flags().StringVarP(&initCmdSrc, "source", "s", "", "The path to the MTA project; the current path is set as the default")
-	initCmd.Flags().StringVarP(&initCmdTrg, "target", "t", "", "The path to the generated Makefile folder; the current path is set as the default")
+	initCmd.Flags().StringVarP(&initCmdSrc, "source", "s", "", "The path to the MTA project; the current path is set as default")
+	initCmd.Flags().StringVarP(&initCmdTrg, "target", "t", "", "The path to the folder in which Makefile is generated; the current path is set as default")
```
Comment (tal-sapan) [inline cmd/init.go]:
```suggestion
	initCmd.Flags().StringVarP(&initCmdTrg, "target", "t", "", "The path to the folder in which the Makefile is generated; the current path is set as default")
```

File: cmd/assembly.go (hunk 4 above)
Comment (allaVolkov) [reply cmd/assembly.go]: done

File: cmd/init.go (hunk 5 above)
Comment (allaVolkov) [reply cmd/init.go]: done

File: cmd/init.go [hunk 6]
```diff
@@ -35,16 +35,16 @@ var buildCmdOutputSync bool
...
 	initCmd.Flags().StringSliceVarP(&initCmdExtensions, "extensions", "e", nil, "The MTA extension descriptors")
 	initCmd.Flags().StringVarP(&initCmdMode, "mode", "m", "", `The mode of the Makefile generation; supported values: "default" and "verbose"`)
 	_ = initCmd.Flags().MarkHidden("mode")
 	initCmd.Flags().BoolP("help", "h", false, `Displays detailed information about the "init" command`)
 
 	// set flags of build command
-	buildCmd.Flags().StringVarP(&buildCmdSrc, "source", "s", "", "The path to the MTA project; the current path is set as the default")
-	buildCmd.Flags().StringVarP(&buildCmdTrg, "target", "t", "", "The path to the results folder; the current path is set as the default")
+	buildCmd.Flags().StringVarP(&buildCmdSrc, "source", "s", "", "The path to the MTA project; the current path is set as default")
+	buildCmd.Flags().StringVarP(&buildCmdTrg, "target", "t", "", `The path to the folder in which MTAR file is created; the path to the "mta_archives" subfolder of the current folder is set as default`)
```
Comment (tal-sapan) [inline cmd/init.go]:
```suggestion
	buildCmd.Flags().StringVarP(&buildCmdTrg, "target", "t", "", `The path to the folder in which the MTAR file is created; the path to the "mta_archives" subfolder of the current folder is set as default`)
```

Comment (allaVolkov) [reply cmd/init.go]: suggestion committed

**Pull request merged**